pkg/kubelet/dockershim/doc.go

```diff
@@ -16,5 +16,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package dockershim does all of the work to implement a contaienr runtime interface
```

Comment:
I would drop "does all of the work to implement" and replace with "package dockershim implements a container runtime interface"

contaienr -> container